pkg/client/active_directory_memberships_object_set.go

```diff
@@ -90,3 +90,93 @@ func buildActiveDirectoryMembershipObjectSet(response interface{}) []*nimbleos.A
 
 	return results
 }
+
+// List of supported actions on object sets
+
+// Remove - Leaves the Active Directory domain.
+func (objectSet *ActiveDirectoryMembershipObjectSet) RemoveObjectSet(id *string, user *string, password *string, force *bool) error {
+
+	removeUri := activeDirectoryMembershipPath
+	removeUri = removeUri + "/" + *id
+	removeUri = removeUri + "/actions/" + "remove"
+
+	payload := &struct {
+		Id       *string `json:"id,omitempty"`
+		User     *string `json:"user,omitempty"`
+		Password *string `json:"password,omitempty"`
+		Force    *bool   `json:"force,omitempty"`
```

Comment:
Are the json tags needed in all of these?

Reply:
These json tags are applicable to only RemoveObjectSet() operation on ActiveDirectoryMembership.

Reply:
No.  The tags are on **every** struct created within the functions generated throughout.  Why is that necessary?

Reply:
please check in another object set, you will not find repeat. thanks

Reply:
Sorry for the confusion.  I didn't mean a repeat of these.  I meant the fact that json tags are present.  I think I figured that out below.